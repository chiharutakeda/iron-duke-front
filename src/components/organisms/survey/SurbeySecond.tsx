import React from 'react';
import { useRouter } from 'next/router';
const SurbeySecond = () => {
  const router = useRouter();
  return (
    <>
      <div>SurbeySecond</div>
      <button
        onClick={() => {
          router.push('SurbeyThird');
        }}
      >
        次へ
      </button>
    </>
  );
};

export default SurbeySecond;
