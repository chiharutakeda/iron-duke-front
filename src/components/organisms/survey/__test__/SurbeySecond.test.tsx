import ReactTestRenderer from 'react-test-renderer';
import SurbeySecond from '../SurbeySecond';

it('SurbeySecond', async () => {
  const tree = await ReactTestRenderer.create(<SurbeySecond />).toJSON();
  await expect(tree).toMatchSnapshot();
});
