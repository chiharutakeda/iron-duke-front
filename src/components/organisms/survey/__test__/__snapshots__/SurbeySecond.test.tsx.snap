// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SurbeySecond 1`] = `
Array [
  <div>
    SurbeySecond
  </div>,
  <button
    onClick={[Function]}
  >
    次へ
  </button>,
]
`;
